### ISSUE TITLE
Ember 2.6 - Uncaught TypeError: Cannot read property 'template' of undefined

### ISSUE THREAD

**Issue by netes** [lightblade/ember-cli-d3#70]
Hey,
I recently updated Ember to 2.6 and there is some compatibility issue. Charts throw an exception during rendering:
`Uncaught TypeError: Cannot read property 'template' of undefined`

It seems that something `ember-cli-d3` use was removed in Ember 2.6. I was trying to find the cause but with no luck :(

**Comment by ming-codes** [comment]
Hi @netes, sorry I don't have much time to investigate this. I would be glad to accept a PR for this.

**Comment by snovity** [comment]
Same here... Will try to find out what is wrong

**Comment by ming-codes** [comment]
I was not able to reproduce this.
<img width="1432" alt="screen shot 2016-06-19 at 10 50 23 am" src="https://cloud.githubusercontent.com/assets/706212/16178949/bf17d2be-360b-11e6-9eaf-4365a21cbb38.png">

**Comment by netes** [comment]
Thanks for reply. I'll try to investigate the issue, maybe it's related to some particular chart parameter or something.

**Comment by snovity** [comment]
@netes which version of jQuery do you use? I'm using jQuery 2.1.4, in @ming-codes example it is jQuery 1.11.3

**Comment by snovity** [comment]
I was able to solve it. I had a component that mixes in `GraphicSupport` and that had no corresponding handlebars file (it worked like this before Ember update). After I created an empty template for that component, it started to work with Ember 2.6.

**Comment by netes** [comment]
@snovity jQuery 2.2.4, but your solution works like a charm. Still - it's weird that we need empty templates for components. I'm wondering where the real reason of this "bug" is.

**Comment by snovity** [comment]
@netes yeah, empty templates should be autogenerated, looks like Ember 2.6 + jQuery 2.x are causing this somehow
